easybuild/easyconfigs/g/GHC/GHC-9.2.2-x86_64.eb

```
@@ -0,0 +1,81 @@
+# This is a binary install that requires a './configure' and 'make install' steps for GHC.
+# We pull the centos7 binary tarball as is the one built against oldest system libs,
+# making it upwards compatible with newer distros.
+#
+# To get a functional 'ghc' binary on the SYSTEM toolchain we need
+# gmp headers and ncurses libtinfo.so.5, to avoid requiring extra OS deps for them
+# we include them in this bundle.
+# Binaries obtained with ghc do not require them, so it should be possible to use this bundle
+# just as builddep among different toolchains.
+#
+# For details, see the PR discussion:
+# https://github.com/easybuilders/easybuild-easyconfigs/pull/11310
+
+easyblock = 'Bundle'
+
+name = 'GHC'
+version = '9.2.2'
+versionsuffix = '-x86_64'
+
+homepage = 'https://haskell.org/ghc/'
+description = """The Glorious/Glasgow Haskell Compiler"""
+
+toolchain = SYSTEM
+
+builddependencies = [
+    ('binutils', '2.32'),
```

@jkuhl-uni the failing test suggests that you have to add `M4` as a build dependency.
```suggestion
    ('binutils', '2.32'),
    ('M4', '1.4.19'),
```

---

Hi @smoors,
sure, I'll push the changes in a few seconds.
Concerning the github integration: I actually used that feature, I think the token I am using is not the right one. I'll keep that in mind next time. Thank you for the link :)